src/Sulu/Component/Content/Document/Structure/PropertyValue.php

```diff
@@ -87,7 +87,7 @@ public function offsetSet($offset, $value)
             return;
         }
 
-        $this->value[$offset] = array_merge($this->value[$offset], $value);
+        $this->value[$offset] = $value;
```

Comment:
@RonRademaker I am a bit afraid of this change... It's definitely not doing the same, and I am not sure if there is some case in which this would break something... Do you need that change already in the 1.4 release? Or would 1.5 be sufficient?